README.md

```diff
@@ -1,128 +1,163 @@
-
-Link to project management tool [Trello](https://trello.com/b/TE5Q9ZYj/t3a2-%F0%9F%8C%B8sakura-pantry).
+## T3A2 Full Stack App
+
+# Sakura Pantry - Japanese Online Grocery Store
+
+## Resources
+
+- [Production site](https://www.google.com.au/)
+- [Back-end repo](https://www.google.com.au/)
+- [Front-end repo](https://www.google.com.au/)
+- [Documentation repo](https://www.google.com.au/)
+- [Trello board](https://trello.com/b/TE5Q9ZYj/t3a2-%F0%9F%8C%B8sakura-pantry)
+
+## Contributors
+
+- [Mana Misumi](https://github.com/Mana12011207)
+- [Irene Nguyen](https://github.com/irenenguyen1017)
+- [Ellen Pham](https://github.com/ellenpham)
+
+# R1 - Website description
+
+## Purpose
+
+We were approached by the owner of Sakura Company, a Japanese food wholesaler, to create an online store for Japanese food products.
+
+The web app serves three main purposes:
+
+- To promote and increase access to Japanese food.
+- To provide a service to the Japanese community in Australia and to those who love Japanese food products.
+- To contribute to business and sales growth for the client, Sakura Company.
+
+### Problem
+
+Prior to having the website, the primary means of obtaining Japanese food was through Japanese supermarkets, mainly located in central areas. This posed geographical constraints for those living in the suburbs. Moreover, limited store numbers and early closing times, often by 5pm, made it challenging for full-time workers to visit. Setting up a physical shop also presented a significant hurdle for Sakura Company, especially with rising rents due to recent unforeseen inflation.
+
+### Solution
+
+The launch of Sakura Pantry is envisioned to address these challenges. Sakura Pantry is an easy-to-use platform for purchasing Japanese food products, accessible to both city dwellers and those in the suburbs, operating 24/7. Unrestricted by the number of shops or closing times, it offers the convenience of online shopping, catering to the busy full-time workforce. By avoiding the need for physical shops, it provides a more cost-efficient means for Sakura Company to offer Japanese food products. In doing so, Sakura Pantry introduces a modern solution to access Japanese food ingredients in response to contemporary demands.
+
+## Functionality/Features
+
+The overall function of the website is to operate as an online store for selling Japanese food products. Below are the MVP features and functionality for the current stage of product development.
+
+### MVP Features
+
+- User authentication and authorisation (Register and Login)
+  - Allow users to create account and login.
+- Password recovery
+  - Allow users to reset password with reset link sent via email.
+- Site navigation and responsive design
+  - Intuitive and responsive UI/UX design for storefront to attract and engage audience.  
+- Search functionality
+  - Allow users to search products by name or by categories.

```

Comment:
Could you please update the label from 'search products by name' to 'search product by keywords'? With this change, users will be able to search for products that match each keyword they type, making it more versatile and relevant.